src/pages/ProfilesExplorerView/components/SceneExploreDiffFlameGraph/components/SceneComparePanel/SceneComparePanel.tsx

```diff
@@ -297,6 +297,8 @@ export class SceneComparePanel extends SceneObjectBase<SceneComparePanelState> {
     const { $timeRange, target } = this.state;
     const { from, to } = $timeRange.state.value;
 
+    this.updateTitle('');
```

Comment:
We also update the panel title to remove any mention to a previously selected preset. E.g. "6h ago:
<img width="174" alt="image" src="https://github.com/user-attachments/assets/9e67faf0-137f-4baa-8cde-1ad483ec6b79">